### DISCUSSION TITLE
Meanings of chief complaint abbreviations in triage table.

### DISCUSSION THREAD

**Discussion by msoren3214** [MIT-LCP/mimic-code#1488] [MIMIC-IV-ED]
How can I access the meanings of abbreviations and other terms used to document chief complaints in MIMIC-ED's triage table--For example, "AAA", "BRBPR", and "transfer"?

**Comment by marymlucas** [comment]
Hi @msoren3214 , because that is a free text field I think you would need some domain knowledge to infer the meanings of the different abbreviations. For example 'BRBPR' usually refers to rectal bleeding (Bright Red Blood Per Rectum), "transfer" likely indicates the patient is being transferred from another facility/hospital, and 'AAA' would likely refer to an abdominal aortic aneurysm. I would cross-check that field with the ICD code for the stay in the diagnoses table.

Mary

**Reply by msoren3214** [reply]
Mary,  Thanks for your helpful response.  Being an ED physician, and having created a table which links unique combinations of patient data, including CCs, to one or more hospital diagnoses, I can likely infer all or most meanings.

**Reply by alistairewj** [reply]
Worth mentioning there was input validation slowly rolled out at the BIDMC, so you'll find that the input is actually fairly well structured if you treat it as a comma delimited field. However, there is still free-text, so it's a messy column!
